docs/Options.md

```diff
@@ -69,16 +69,19 @@ By default, a username is shown only when it's not the same as `$LOGNAME`, when
 
 ### Directory \(`dir`\)
 
-Directory is always shown and truncated to the value of `SPACEFISH_DIR_TRUNC`. While you are in repository, it shows only root directory and folders inside it.
+Directory is always shown and truncated to the value of `SPACEFISH_DIR_TRUNC`. While you are in repository, it shows only root directory and folders inside it. If current directory is write-protected or if current user has not enough rights to write in it, a padlock (by default) is displayed as a suffix.
```

Comment:
```suggestion
Directory is always shown and truncated to the value of `SPACEFISH_DIR_TRUNC`. While you are in Git repository, this section shows only the project title and relevant status icons. If current directory is write-protected or if current user doesn't have write permissions, a padlock (by default) will be displayed as a suffix.
```

Reply:
This paragraph of text doesn't make much sense and could be quite confusing, here's a quick syntactic clean-up.